tox.ini

```diff
@@ -1,19 +1,27 @@
 [tox]
 minversion=2.3.1
 envlist =
-    py{27,35}-ansible22-{pylint,unit,flake8,yamllint,generate_validation}
+    py{27,35}-ansible22-{lint,unit, functional}
 skipsdist=True
 skip_missing_interpreters=True
 
 [testenv]
 deps =
     -rtest-requirements.txt
-    py35-flake8: flake8-bugbear
-    ansible22: ansible~=2.2
+    py35-lint: flake8-bugbear
+    ansible22: ansible==2.2.0.0
 
 commands =
-    flake8: flake8
-    pylint: python setup.py lint
-    yamllint: python setup.py yamllint
+    lint: flake8
+    lint: python setup.py lint
+    lint: python setup.py yamllint
+    lint: python setup.py generate_validation
     unit: nosetests
-    generate_validation: python setup.py generate_validation
+    py27-functional: python setup.py molecule_tests
+    py35-functional: echo "no py35 functional tests currently"
```

Comment:
For now, yes. I haven't had a chance to contribute changes to make it py35 compatible. Even then, I suspect we might hit some Ansible issues, since they are still fixing some remaining py3 compat issues.